aquacomputer_d5next.c

```diff
@@ -621,11 +664,15 @@ struct aqc_data {
 	u32 power_cycles;
 
 	/* Sensor values */
-	s32 temp_input[20];	/* Max 4 physical and 16 virtual or 8 physical and 12 virtual */
-	s32 speed_input[8];
-	u32 speed_input_min[8];
+	/*
+	 * temp_input has a maximum of 4 physical + 16 virtual + 20 aquabus,
+	 * or 8 physical + 12 virtual + 20 aquabus, depending on the device
+	 */
```

Comment:
Do you think I should combine this multi-line comment with the `/* Sensor values */` heading above?

Reply:
OK should be fixed now. Thanks!